src/db/modelRegistration.ts

```diff
@@ -31,4 +32,5 @@ export function registerModels(): void {
   registerModel(OwnerUpdateRequestedEventModel);
   registerModel(AddressDriverSplitReceiverModel);
   registerModel(AccountMetadataEmittedEventModel);
+  registerModel(GivenEventModel);
```

Comment:
Just a suggestion so that I can sleep at nights:

Mind if we add the registrations as a 🌲
*
**
\***
\****
by line length?

🤓